### ISSUE TITLE
Probably should list out chart types that are supported on README

### ISSUE THREAD

**Issue by andyd0** [gtnx/pandas-highcharts#50]
Like column charts are not.  Package is easy to use given the dataframe support but would save time to know what chart types are actually supported

**Comment by dirkesquire** [comment]
I agree. Also spline charts don't seem supported either. I would really like to draw a line chart with smoothed curves.

e.g.: https://api.highcharts.com/highcharts/plotOptions.spline
